SECURITY.md

```diff
@@ -0,0 +1,45 @@
+# Security Policy
+
+## What is considered a vulnerability?
+
+There are some types of [low-severity][severity] vulnerabilities that we will not acknowledge as CVE and treat as bugs instead. All vulnerabilities with a severity of medium and above will of course be acknowledged and fixed.
+
+Please see the below section on how we treat [ReDoS] vulnerabilities.
+
+If you are unsure whether a vulnerability you found qualifies, please report it as a vulnerability via email (see below).
+
+### ReDoS
+
+Prism is a regex-based syntax highlighter. As such, the main types of vulnerabilities reported to us are [ReDoS] vulnerabilities ([CWE-1333](https://cwe.mitre.org/data/definitions/1333.html)), aka slow regexes.
+
+However, not all ReDoS is created equal. A slow regex can be have a [worst-case time complexity](https://en.wikipedia.org/wiki/Time_complexity) anywhere from _O(n<sup>2</sup>)_ to  _2<sup>O(n)</sup>_. This matters because a worst-case time complexity _≥ O(n<sup>3</sup>)_ is a [high severity][severity] vulnerability while _O(n<sup>2</sup>)_ is low or medium severity in the context of Prism.
```

Comment:
```suggestion
However, not all ReDoS is created equal.
A slow regex can be have a [worst-case time complexity](https://en.wikipedia.org/wiki/Time_complexity) anywhere from _O(n<sup>2</sup>)_ to  _2<sup>O(n)</sup>_.
This matters because a worst-case time complexity _≥ O(n<sup>3</sup>)_ is a [high severity][severity] vulnerability while _O(n<sup>2</sup>)_ is low or medium severity in the context of Prism.
```